### ISSUE TITLE
Cannot export to Excel

### ISSUE THREAD

**Issue by DaniilMaslyuk** [DaxStudio/DaxStudio#688]
Error message: 29-Jul-21 14:54:11		Could not load file or assembly 'System.Diagnostics.Tracing, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

![image](https://user-images.githubusercontent.com/10683126/127454069-abc7767e-c27c-49be-af2b-4e1ceb4ee406.png)

**Comment by dgosbell** [comment]
Thanks for logging this. Can you hold down the shift key while starting DAX Studio to enable the [diagnostic logging](https://daxstudio.org/documentation/troubleshooting/logging-settings/) then re-produce this issue and then close DAX Studio and upload the log file to this issue on github. The diagnostic log should give me a call stack that should help isolate this issue.

**Comment by DaniilMaslyuk** [comment]
[DaxStudio-20210801.log](https://github.com/DaxStudio/DaxStudio/files/6911951/DaxStudio-20210801.log)

**Comment by dgosbell** [comment]
Sorry, @DaniilMaslyuk but somehow the diagnostic logging did not kick in and that log captured nothing. You will see a Windows notification message when the diagnostic logging is enabled due to the shift key being held down. Can you maybe try again? 

And could you maybe try using the portable version to see if that is any different. The file that the error is complaining about is one of the ones distributed with DAX Studio so I can't think why it would have trouble loading it

Also there are actually 4 different ways of exporting to Excel:
* Static Excel - when launched from the Excel addin
* Linked Excel  - when launched from the Excel addin
* Static Excel - when launched stand alone or from external tools
* Linked Excel  - when launched stand alone or from external tools

Which one of these is producing the error for you?

**Comment by DaniilMaslyuk** [comment]
Tried again, and this time I got the Windows notification :-)
[DaxStudio-20210802.log](https://github.com/DaxStudio/DaxStudio/files/6913827/DaxStudio-20210802.log)

On exporting to Excel: isn't there a way to export to Excel even after launching DAX Studio by itself? I thought it was a feature introduced in 2.13?
![image](https://user-images.githubusercontent.com/10683126/127792505-f7bca87a-5005-4487-9871-eeef01a96931.png)

Anyway, for static Excel, when launched
- From Excel: success
- Standalone: fail
- As an externa tool: fail
Linked Excel worked even when I launched a standalone DAX Studio instance, so I didn't test further.
